### PR TITLE
Dev card echaoo

### PR DESCRIPTION
**Pull request by echaoo**
card component dev finished, plz code review. @zzuu666

### CODE REVIEW

**Comment by sinchang** [comment]
[不需要处理父级class与style](https://github.com/zzuu666/antue/blob/master/docs/rules/component.zh-CN.md#不需要处理父级class与style)

File: components/card/card.vue [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        </p>
+      </div>
+      <slot v-if="!loading"></slot>
+    </div>
+  </div>
+</template>
+
+<script>
+  export default {
+    data () {
```
Comment (zzuu666) [inline components/card/card.vue]: `name` is required.

File: components/card/card.vue [hunk 2]
```diff
@@ -0,0 +1,86 @@
+<template>
+  <div
+    :class="classes"
+    ref=card>
```
Comment (zzuu666) [inline components/card/card.vue]: 1. If need ref the correct declare is `ref="card"`
2. `ref` seem like is useless.

File: components/card/card.vue [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    </div>
+  </div>
+</template>
+
+<script>
+  export default {
+    data () {
+      return {
+        widerPadding: false,
+        updateWiderPaddingCalled: true
```
Comment (zzuu666) [inline components/card/card.vue]: `widerPadding` & `updateWiderPaddingCalled` in Ant Design is useful. But them is changeless in here.

File: components/card/card.vue [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+          this.updateWiderPaddingCalled ? `${this.prefixCls}-padding-transition` : ''
+        ]
+      }
+    }
+  }
+</script>
+
+<style>
+
+</style>
```
Comment (zzuu666) [inline components/card/card.vue]: Should remove.

File: components/card/grid.vue [hunk 5]
```diff
@@ -0,0 +1,16 @@
+<template>
+  <div :class="`${prefixCls}-grid`">
+    <slot></slot>
+  </div>
+</template>
+
+<script>
+  export default {
+    props: {
```
Comment (zzuu666) [inline components/card/grid.vue]: `name` is required.

File: components/card/index.js [hunk 6]
```diff
@@ -0,0 +1,10 @@
+/**
+ * Created by yiche on 17/9/24.
```
Comment (zzuu666) [inline components/card/index.js]: Sure? 😂

**Pull request merged**